controllers/contact-controller.js

```diff
@@ -170,6 +170,7 @@ function convertContactsToSearchResults(contacts) {
       const categories = 'TBD';
```

Comment:
This should no longer be TBD.  We can either return the actual categories here, or drop this and pull the categories out of the details.

Reply:
These overview fields were created with the intention to facilitate `ContactPreview`. I've seen we're reading tags like this in `ContactPreview`: `const [tag1, tag2, tag3] = contact.tags;`. This `categories` field should be exactly this `tags` field.
I'd rename `caregories` to `tags` and return the intended tag array.